src/blocks/slider-item/edit.js

```diff
@@ -42,6 +43,39 @@ export default function SliderItemEdit(props) {
 	} = attributes;
 
 	useEffect(() => {
+		// 既に設定されているスタイルを優先し、未設定の場合のみ新しい値を計算して適用
+		const existingPadding = style?.spacing?.padding;
+		let newPadding;
+	
+		if (!existingPadding) {  // spacing.paddingが未設定の場合のみ新しい値を設定
+			switch (padding_left_and_right) {
+				case '0px':
```

Comment:
@mtdkei ここはただの識別用数値なので px はつかないのでは？
```suggestion
				case '0':
```